quart_redis/__init__.py

```diff
@@ -46,7 +48,7 @@ async def init_redis():
 
         @app.after_serving
         async def close_redis():
-            await RedisHandler._connection.close()
+            RedisHandler._connection.close()
```

Comment:
The docs say it's an awaitable: [example](https://redis-py.readthedocs.io/en/stable/examples/asyncio_examples.html#Connecting-and-Disconnecting)